[1.7.2] marketplace access bonus (#6728)
vcmi/vcmi: docs/modders/Bonus/Bonus_Types.md

### UNIT_DEFENDING

Bonus that is automatically granted to unit whenever unit uses defend action in battle. Has no gameplay effects, however mods can use presence of this bonus to provide abilities that are active while unit is defending using limiter

### MARKETPLACE_ACCESS

> Increases amount of counted marketplaces (in town or in adventure map tavern). You may want to use PLAYER_PROPAGATOR with this bonus to make its effect player wide.

Reviewer: Copy-paste leftover?
```suggestion
Increases amount of counted marketplaces when trading in town. You may want to use PLAYER_PROPAGATOR with this bonus to make its effect player wide.
```
